packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -10,6 +10,8 @@ import {
   spaceScaledXxxs,
 } from '@cloudscape-design/design-tokens';
 import { ContentLayout } from '@cloudscape-design/components';
+import messages from '@cloudscape-design/components/i18n/messages/all.all';
+import { I18nProvider } from '@cloudscape-design/components/i18n';
```

Comment:
places provider here because we export both a View and Edit Dashboard component and they both implement `InternalDashboard`